libs/fluxc/src/main/java/org/wordpress/android/fluxc/model/list/ListOrder.kt

```diff
@@ -7,7 +7,7 @@ enum class ListOrder(val value: String) {
     DESC("DESC");
     companion object {
         fun fromValue(value: String): ListOrder? {
-            return values().firstOrNull { it.value.toLowerCase(Locale.ROOT) == value.toLowerCase(Locale.ROOT) }
+            return values().firstOrNull { it.value.lowercase(Locale.ROOT) == value.lowercase(Locale.ROOT) }
```

Comment:
```suggestion
            return values().firstOrNull { it.value.lowercase() == value.lowercase() }
```

As these `lowercase`s here and in other places are not for meant for user UI, I'd suggest removing `Locale` to simplify, as it shouldn't matter. Also, this is quite fragile: we don't want to map enums differently, depending on locale of the device.

Reply:
@wzieba Just so I'm clear, are you suggesting removing `Locale.ROOT` from every place I changed `toLowerCase` to `lowercase` in this PR?

Reply:
Yes, you're correct 👍  Also usage of `Locale.US`, as I think I've seen it in this PR as well.

Reply:
Done in [ca1d061](https://github.com/wordpress-mobile/WordPress-Android/pull/21515/commits/ca1d0617d07b424ba85be12564bac57f70878258)